lib/reline/line_editor.rb

```diff
@@ -1161,8 +1162,12 @@ def save_old_buffer
   end
 
   def push_past_lines
-    if @old_buffer_of_lines != @buffer_of_lines
-      @past_lines.push([@old_buffer_of_lines, @old_byte_pointer, @old_line_index])
```

Comment:
Looks like `@old_byte_pointer` and `@old_line_index` are not used anymore. We can remove initialization code from `def save_old_buffer`.

Reply:
I've removed initialization code. Thank you.

[Deleted unused variables: @old_byte_pointer and @old_line_index](https://github.com/ruby/reline/pull/707/commits/95a7108a0de765549e480a60f226636fcf7069d0)